js/lib/node-artist/rich-text-processor.js

```diff
@@ -0,0 +1,21 @@
+import parser from "../../lib/rich-text/parser.js";
+
+function mapContent(data, element, expression) {
+
+    const parsed = parser(data[expression]);
+    element.appendChild(parsed);
+
+}
+
+export default function(node, data) {
```

Comment:
I thought we had agreed to name the default functions, for consistency?

Reply:
Thanks fixed